packages/api/test/unit/client/fetch.test.ts

```diff
@@ -116,12 +116,16 @@ describe("FetchError", function () {
         );
       }
 
-      await expect(fetch(url, {signal: signalHandler?.()})).to.be.rejected.then((error: FetchError) => {
-        expect(error.type).to.be.equal(testCase.errorType);
-        expect(error.code).to.be.equal(testCase.errorCode);
+      await expect(fetch(url, {signal: signalHandler?.()})).rejects.toSatisfy((err) => {
```

Comment:
can't this be changed to?
```suggestion
      await expect(fetch(url, {signal: signalHandler?.()})).rejects.toSatisfy((err: FetchError) => {
```

Reply:
Not that's the strict type issue we can't assign type inside the function descriptor. 

```
Argument of type '(err: FetchError) => true' is not assignable to parameter of type '(value: unknown) => boolean'
```